jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -387,6 +397,22 @@ class CodeWhispererCodeScanManager(val project: Project) {
         project.refreshDevToolTree()
     }
 
+    private fun sendCodeScanTelemetryToServiceAPI(
+        project: Project,
+        programmingLanguage: CodeWhispererProgrammingLanguage?,
+        codeScanJobId: String?
+    ) {
+        runIfIamIdentityCenterConnection(project) {
+            try {
+                val response = CodeWhispererClientAdaptor.getInstance(project)
+                    .sendCodeScanTelemetry(programmingLanguage, codeScanJobId)
+                LOG.debug { "Successfully sent code scan telemetry. RequestId: ${response.responseMetadata().requestId()}" }
+            } catch (e: Exception) {
+                LOG.debug(e) { "Failed to send code scan telemetry." }
```

Comment:
nit:log requestId here?

Reply:
Added expliclty logging the requestId (though I think LOG.debug(e) will also log everything?)

Reply:
I see, in that case this is finee